libs/utils/energy.py

```diff
@@ -34,6 +34,10 @@
         'conf' : {
             'sites' : [ 'A7 Jcore', 'A15 Jcore' ],
             'kinds' : [ 'energy']
+        },
+        'channel_map' : {
+            'little' : 'A7 Jcore',
```

Comment:
Kind of boring, sorry... but the current libs/utils/results.py has an hardcoded "LITTLE" value.
We should keep that script working at least for the eas/rfc test.

I guess we can either rename these channels maps, or add a patch to this series which update the previous script as well.